compose/tests/test_docker_compose.py

```diff
@@ -123,3 +130,4 @@ def test_can_exec_commands():
         assert result[0] == 'my_test\n', "The echo should be successful"
         assert result[1] == '', "stderr should be empty"
         assert result[2] == 0, 'The exit code should be successful'
+"""
```

Comment:
```suggestion
```

Not needed to comment if we skip (I hope).

Reply:
resolving this to continue discussion where the pytest.mark.skip is suggested